### PR TITLE
{Find} az find: Fix Aladdin Service URL

### PR DESCRIPTION
**Pull request by mirdaki**
**Description**  

Accidentally pushed the wrong URL to the `az find` module. The current one will fail.

**Testing Guide**  

N/A

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
az find

**Review by houk-ms — APPROVED** [review]

**Review by yungezz — APPROVED** [review]

**Comment by yungezz** [comment]
hi @mirdaki is it ok to merge it now? @houk-ms will merge it then.

**Review by jiasli — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/find/custom.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def call_aladdin_service(query):
     if telemetry_core.is_telemetry_enabled() and subscription_id is not None:
         context['subscriptionId'] = subscription_id
 
-    api_url = 'https://aladdindevwestus-app.azurewebsites.net/api/v1.0/examples'
+    api_url = 'https://app.aladdin.microsoft.com/api/v1.0/examples'
```
Comment (jiasli) [inline src/azure-cli/azure/cli/command_modules/find/custom.py:118]: Will this work in air-gapped cloud? Perhaps it should go here 

https://github.com/Azure/azure-cli/blob/4055456f27f756bebe6e042baaf2396eb4154e47/src/azure-cli-core/azure/cli/core/cloud.py#L358-L362

to make it customizable.

Comment (yungezz) [reply src/azure-cli/azure/cli/command_modules/find/custom.py:118]: this is not working on air-gapped clouds. In air-gapped cloud, the solution is offline pacakage now. Once the service available in air-gapped clouds, we'll need register it in cloud.py.

**Review by fengzhou-msft — APPROVED** [review]

**Pull request merged**